include/SDL3/SDL_stdinc.h

```diff
@@ -1024,7 +1024,8 @@ extern SDL_DECLSPEC void SDLCALL SDL_aligned_free(void *mem);
 /**
  * Get the number of outstanding (unfreed) allocations.
  *
- * \returns the number of allocations.
+ * \returns the number of allocations or
+ *          -1 if allocation counting is disabled.
```

Comment:
I wrote -1 here, should we generalize it to negative values?

Reply:
No, -1 is correct and fine.